src/services/auth/plugins/password/index.test.ts

```diff
@@ -745,3 +745,43 @@ describe('GET members current password status', () => {
     expect(response.json()).toEqual({ hasPassword: true });
   });
 });
+
+describe('Regression tests', () => {
```

Comment:
Should this be named differently ? 

`Flow tests` maybe instead of singling it out as a `regression test` ?